usaspending_api/references/migrations/0052_toptieragencypublisheddabsview.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 2.2.19 on 2021-03-11 18:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("references", "0051_auto_20210223_1404"),
+        ("submissions", "0015_submissionattributes_history"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ToptierAgencyPublishedDABSView",
+            fields=[
+                ("toptier_code", models.TextField()),
+                ("name", models.TextField()),
+                ("abbreviation", models.TextField()),
+                (
+                    "toptier_agency",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        primary_key=True,
+                        related_name="toptieragencypublisheddabsview",
+                        serialize=False,
+                        to="references.ToptierAgency",
+                    ),
+                ),
+                ("user_selectable", models.BooleanField()),
+            ],
+            options={
+                "db_table": "vw_published_dabs_toptier_agency",
+                "managed": False,
+            },
+        ),
+        migrations.RunSQL(
+            sql=[
+                """
+                CREATE VIEW vw_published_dabs_toptier_agency AS
+                SELECT
+                    DISTINCT ON (ta.toptier_code)
+                    ta.toptier_code,
+                    ta.name,
+                    ta.abbreviation,
+                    ta.toptier_agency_id,
+                    a.id AS agency_id,
```

Comment:
Should there be an entry in `fields` above for `agency`?

Reply:
Not sure how that was missed. Honestly I'm debating wether any items need to be in fields since Django isn't managing the model since it is a view. I'll correct it one way or another

Reply:
Removing the items in the fields list seems OK